pkg/store/sandbox/netns_windows.go

```diff
@@ -63,6 +64,11 @@ func (n *NetNS) Remove() error {
 	if n.restored {
 		n.restored = false
 	}
+	hcnNamespace, err := hcn.GetNamespaceByID(n.path)
+	if err != nil {
```

Comment:
You might want to handle the `GetNamespaceByID` returning `NotFound`. This is also a success for `NetNS` based on the comments.